floris/simulation/flow_field.py

```diff
@@ -36,6 +42,7 @@ class FlowField(FromDictMixin):
     turbulence_intensity: float = field(converter=float)
     reference_wind_height: float = field(converter=float)
     time_series : bool = field(default=False)
+    het_config: dict = field(default=None)
```

Comment:
I think `het_config` is yet obscure if you don't already know that this is a thing. Instead, I propose to change the name to something like `speed_ups_map`. @bayc @misi9170 @paulf81 what do you think?

Reply:
Agree that this is a more descriptive name and will more clearly indicate its purpose, I say switch it

Reply:
My only reservation is that we use the term "heterogeneous" pretty regularly throughout the code, so I kind of like having some indication that this is related to heterogeneity in the name. What do you think about `heterogeneous_speed_ups_map`? A little long and clunky...

Reply:
@misi9170 I like it. If we support heterogeneous wind directions, that will logically fit into `heterogenous_inflow_config` also. I think `x/y_loc` should be simply `x/y` since the `_loc` doesn't add much clarity. It's reasonably obvious that these are the locations of the speed multipliers.

Here's where we're at:
```python
heterogenous_inflow_config = {
    "speed_multipliers": [1.0, 2.0, 3.0],
    "x": [ -300.0, 300.0, -300.0, 300.0],
    "y": [ -300.0, 300.0, -300.0, 300.0]
}